.github/workflows/image_post.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run this workflow on pull requests
+name: image_post
```

Comment:
I don't agree with having two different workflows here.

I have created https://github.com/NVIDIA/k8s-device-plugin/pull/483/files#diff-b4df0a4f0d80f73138c476afbd7aefdac9df339642ddfba323d27c8cbabb92e2 which sets `PUSH_ON_BUILD` depending on whether a PR is merged or not.

That PR also splits the commits into smaller chunks to make it simpler to review.